cloud/pkg/synccontroller/objectsync.go

```diff
@@ -71,6 +93,12 @@ func (sctl *SyncController) manageObject(sync *v1alpha1.ObjectSync) {
 
 func sendEvents(err error, nodeName string, sync *v1alpha1.ObjectSync, resourceType string,
 	objectResourceVersion string, obj interface{}) {
+	// some workaround to keep the CRD resource type consistent with the edge side
+	// TODO: investigate to have a more systematic way to handle.
+	if resourceType == "missions" {
+		resourceType = "mission"
```

Comment:
one is singular and the other is plural. is this by design?

Reply:
sync offline.